packages/ia-components/sandbox/details/NavWrap.jsx

```diff
@@ -88,10 +88,10 @@ class NavSearchLI extends React.Component {
     this.setState({ value: event.target.value });
   }
 
-  onClick(unusedEvent) {
+  onClick(ev) {
     // this.formElement.submit(); // TODO Doesnt work - not sure why
     this.onSubmit(unusedEvent);
-    return false;
+    ev.preventDefault(); // Dont propogate event
```

Comment:
pinging again to ask the question - do we need to stop event propagation?

Reply:
In this case no - though as I said, I thought it was good practice to catch clicks you've acted on. change made